### ISSUE TITLE
[RESOLVED] Current version not working

### ISSUE THREAD

**Issue by FrontMage** [nadeesha/saul#15]
I've done as the readme says, but it does not test any file.

Results are like these:

> saul@0.1.0 test /home/xinbg/Documents/pincloud.supe
> nyc mocha src/**/*.test.js node_modules/.bin/saul

Warning: Could not find any test files matching pattern: src/**/*.test.js

  0 passing (1ms)

----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
All files |  Unknown |  Unknown |  Unknown |  Unknown |                |
----------|----------|----------|----------|----------|----------------|

---
Also wondering how to exclude node_modules folder?

**Comment by nadeesha** [comment]
@FrontMage can you paste the content of your `.saulrc` file please?

**Comment by FrontMage** [comment]
package.json
```
{
  "name": "Neon",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "test": "nyc mocha node_modules/.bin/saul"
  },
  "dependencies": {
    "mocha": "^3.4.1",
    "nyc": "^10.3.2",
    "saul": "^0.2.4"
  },
  "nyc": {
    "include": [
      "webapps/**/*.js"
    ]
  }
}
```
.saulrc
```
{
    "fileGlob": "webapps/**/*.js",
    "customEnginesDir": "./src/custom-saul-engines"
}
```
test.js in webapps/controllers/test.js
```
// @t "can sum" sum(1, 2) equals 3
// @t "can handle zeroes" sum(0, 0) equals 0
exports.sum = function sum(a, b) {
  return a + b;
};
```
By the way, I found it working using `"saul": "0.0.17",`@nadeesha

**Comment by nadeesha** [comment]
Thanks @FrontMage!

I don't see anything wrong with this. If you're not using any custom engines, you don't need `custom-saul-engines` key in .saulrc.

I've just released `v0.2.4`. Can you upgrade to that and check whether the problem is fixed?

**Comment by FrontMage** [comment]
Removed custom-saul-engines, upgrade to v0.2.4 outputs these:
```
➜ npm test

> Neon@0.0.1 test /home/xinbg/Documents/pincloud.supe
> nyc mocha node_modules/.bin/saul



  0 passing (1ms)

----------|----------|----------|----------|----------|----------------|
File      |  % Stmts | % Branch |  % Funcs |  % Lines |Uncovered Lines |
----------|----------|----------|----------|----------|----------------|
All files |  Unknown |  Unknown |  Unknown |  Unknown |                |
----------|----------|----------|----------|----------|----------------|
```
It seems not finding my test files.

**Comment by nadeesha** [comment]
My apologies. Since `0.2.0`, we've introduced a `~` character in front of the engine name.

So, the correct format of the test should be:

```js
// @t "can sum" sum(1, 2) ~equals 3
```

This is my mistake as I've overlooked to update certain parts of the docs and the demo. But I've made the updates with #18 

If you change the format of the tests, it should work with the latest version.  I'll try to have some kind of a warning for tests written in the deprecated method. Thanks for reporting this @FrontMage. Let me know if I can a bit more on this.
